[2024-11-25T15:55:42.782+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_project.train_model manual__2024-11-25T15:55:24.597952+00:00 [queued]>
[2024-11-25T15:55:42.809+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_project.train_model manual__2024-11-25T15:55:24.597952+00:00 [queued]>
[2024-11-25T15:55:42.810+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 4
[2024-11-25T15:55:42.831+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): train_model> on 2024-11-25 15:55:24.597952+00:00
[2024-11-25T15:55:42.857+0000] {standard_task_runner.py:60} INFO - Started process 1371 to run task
[2024-11-25T15:55:42.862+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ml_project', 'train_model', 'manual__2024-11-25T15:55:24.597952+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/ml_project.py', '--cfg-path', '/tmp/tmpc9ypspzf']
[2024-11-25T15:55:42.865+0000] {standard_task_runner.py:88} INFO - Job 21: Subtask train_model
[2024-11-25T15:55:42.923+0000] {task_command.py:423} INFO - Running <TaskInstance: ml_project.train_model manual__2024-11-25T15:55:24.597952+00:00 [running]> on host e084ca88f62c
[2024-11-25T15:55:43.008+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ml_project' AIRFLOW_CTX_TASK_ID='train_model' AIRFLOW_CTX_EXECUTION_DATE='2024-11-25T15:55:24.597952+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-25T15:55:24.597952+00:00'
[2024-11-25T15:55:43.015+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/huggingface_hub/file_download.py:797 FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
[2024-11-25T15:55:46.943+0000] {logging_mixin.py:188} WARNING - Map:   0%|          | 0/35000 [00:00<?, ? examples/s]
[2024-11-25T15:56:13.871+0000] {logging_mixin.py:188} WARNING - Map: 100%|##########| 35000/35000 [00:26<00:00, 1299.87 examples/s]
[2024-11-25T15:56:14.083+0000] {logging_mixin.py:188} WARNING - Map: 100%|##########| 35000/35000 [00:27<00:00, 1289.74 examples/s]
[2024-11-25T15:56:14.099+0000] {logging_mixin.py:188} WARNING - Map:   0%|          | 0/15000 [00:00<?, ? examples/s]
[2024-11-25T15:56:18.319+0000] {logging_mixin.py:188} WARNING - Map: 100%|##########| 15000/15000 [00:04<00:00, 3554.50 examples/s]
[2024-11-25T15:56:18.420+0000] {logging_mixin.py:188} WARNING - Map: 100%|##########| 15000/15000 [00:04<00:00, 3473.01 examples/s]
[2024-11-25T15:56:18.426+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/huggingface_hub/file_download.py:797 FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
[2024-11-25T15:56:19.467+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/pydantic/_internal/_fields.py:149 UserWarning: Field "model_server_url" has conflict with protected namespace "model_".

You may be able to resolve this warning by setting `model_config['protected_namespaces'] = ()`.
[2024-11-25T15:56:19.484+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/pydantic/_internal/_config.py:321 UserWarning: Valid config keys have changed in V2:
* 'schema_extra' has been renamed to 'json_schema_extra'
[2024-11-25T15:56:20.065+0000] {logging_mixin.py:188} WARNING - 2024/11/25 15:56:20 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet
[2024-11-25T15:56:20.405+0000] {logging_mixin.py:188} WARNING -   0%|          | 0/3282 [00:00<?, ?it/s]
[2024-11-25T15:56:23.664+0000] {logging_mixin.py:188} WARNING -   0%|          | 1/3282 [00:03<2:58:07,  3.26s/it]
[2024-11-25T15:56:26.874+0000] {logging_mixin.py:188} WARNING -   0%|          | 2/3282 [00:06<2:56:31,  3.23s/it]
[2024-11-25T15:56:30.290+0000] {logging_mixin.py:188} WARNING -   0%|          | 3/3282 [00:09<3:01:08,  3.31s/it]
[2024-11-25T15:56:33.376+0000] {logging_mixin.py:188} WARNING -   0%|          | 4/3282 [00:12<2:56:10,  3.22s/it]
[2024-11-25T15:56:35.884+0000] {logging_mixin.py:188} WARNING -   0%|          | 5/3282 [00:15<2:41:58,  2.97s/it]
[2024-11-25T15:56:38.421+0000] {logging_mixin.py:188} WARNING -   0%|          | 6/3282 [00:18<2:33:58,  2.82s/it]
[2024-11-25T15:56:41.928+0000] {logging_mixin.py:188} WARNING -   0%|          | 7/3282 [00:21<2:46:11,  3.04s/it]
[2024-11-25T15:56:45.195+0000] {logging_mixin.py:188} WARNING -   0%|          | 8/3282 [00:24<2:50:00,  3.12s/it]
[2024-11-25T15:56:48.851+0000] {logging_mixin.py:188} WARNING -   0%|          | 9/3282 [00:28<2:59:10,  3.28s/it]
[2024-11-25T15:56:51.573+0000] {logging_mixin.py:188} WARNING -   0%|          | 10/3282 [00:31<2:49:38,  3.11s/it]
[2024-11-25T15:56:54.133+0000] {logging_mixin.py:188} WARNING -   0%|          | 11/3282 [00:33<2:40:22,  2.94s/it]
[2024-11-25T15:56:57.631+0000] {logging_mixin.py:188} WARNING -   0%|          | 12/3282 [00:37<2:49:34,  3.11s/it]
[2024-11-25T15:57:00.722+0000] {logging_mixin.py:188} WARNING -   0%|          | 13/3282 [00:40<2:49:11,  3.11s/it]
[2024-11-25T15:57:04.927+0000] {logging_mixin.py:188} WARNING -   0%|          | 14/3282 [00:44<3:07:12,  3.44s/it]
[2024-11-25T15:57:07.729+0000] {logging_mixin.py:188} WARNING -   0%|          | 15/3282 [00:47<2:56:44,  3.25s/it]
[2024-11-25T15:57:10.605+0000] {logging_mixin.py:188} WARNING -   0%|          | 16/3282 [00:50<2:50:37,  3.13s/it]
[2024-11-25T15:57:13.135+0000] {local_task_job_runner.py:234} INFO - Task exited with return code -9
[2024-11-25T15:57:13.447+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
